src/Apps/Artist/routes.tsx

```diff
@@ -81,7 +81,7 @@ export const routes: RouteConfig[] = [
             $for_sale: Boolean
             $sort: String
             $at_auction: Boolean
-            $ecommerce: Boolean
```

Comment:
@mzikherman are `ecommerce` and `acquireable` interchangeable?

Reply:
No, `acquireable` is the correct one to use. I had called it `ecommerce` but then realized that something could be 'enabled for ecommerce', but not actually able to be bought (it's on hold, etc.), so renamed. There isn't an `ecommerce` anymore.